server/models/imageUpload.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const timeSlot = new mongoose.Schema({ start: Date, end: Date });
+
+const imageUploadSchema = new mongoose.Schema({
+  imageUrlPath: {
+    type: String,
+    default: "",
+    },
+});
+
+// export default imageUpload;
```

Comment:
I do not about this but @moffatethan  my put comment here, I would suggest you remove the un wanted comments from your code to make it look cleaner!, as he suggested to me earlier.